recipes/wbgapi/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "wbgapi" %}
+{% set version = "1.0.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: cd04c75830656ec4fb08b5061e06da9ac08b210e2f5add8217a173ef05223ba0
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.0
+    - pip
+    - requests
+    - PyYAML
+    - tabulate
```

Comment:
It seems like this is only required when running the package, not when building it.

Ref: https://github.com/tgherzog/wbgapi/blob/a0f8658b7a74ec79256d7b66ff58cb95726e89aa/setup.py#L28

Reply:
I originally did not include them but the build kept failing, which is why I added them, but will double check.

Reply:
Hmm, strange. Do you have the logs for that commit?

Reply:
Sadly can't find the azure build log anymore. Will try it without the host requirements again and we ll se if it fails.

Reply:
Okay I think I found the reason for the issue.
`  Traceback (most recent call last):
    File "<string>", line 2, in <module>
    File "<pip-setuptools-caller>", line 34, in <module>
    File "/home/conda/staged-recipes/build_artifacts/wbgapi_1647278866514/work/setup.py", line 5, in <module>
      from wbgapi.__version__ import __version__ as pkgVersion
    File "/home/conda/staged-recipes/build_artifacts/wbgapi_1647278866514/work/wbgapi/__init__.py", line 9, in <module>
      import requests
  ModuleNotFoundError: No module named 'requests'
  error: subprocess-exited-with-error
`
Don't know why for certain because it is not my repository, but because they import version from wbgapi it requires all the packages imported to wbgapi, so basically all needed for running the library.

Reply:
Ahh yes, that makes sense – interesting. Ok in that case you'll need to keep it. But I would suggest opening an issue upstream about this because it's a strange setup.

Reply:
Will post an issue on the original repo about it. Thank you for your time.